docs/index.md

```diff
@@ -25,7 +25,77 @@ layout: home
         >
       </div>
     </div>
-    <CodeToggler class="hero-toggler" />
+    <div class="hero-toggler">
+
+<SnippetToggler initial="REST" :choices="['REST', 'GraphQL', 'JS-SDK']" label="API">
```

Comment:
Is there ever a case where the first item in `choices` shouldn't be the `initial`?

Reply:
& Lets just call it `SDK` instead of `JS-SDK`